README.md

```diff
@@ -735,3 +737,75 @@ folio.setReporters([
   new folio.reporters.junit({ outputFile: 'results.xml' })
 ]);
 ```
+
+## Expect
+
+### Add custom matchers using expect.extend
+
+Folio uses [expect](https://jestjs.io/docs/expect) under the hood which has the functionality to extend it with [custom matchers](https://jestjs.io/docs/expect#expectextendmatchers). See the following example where a custom `toBeWithinRange` function gets added.
+
+<details>
+  <summary>folio.config.ts</summary>
+
+```ts
+import * as folio from 'folio';
+
+folio.setConfig({ testDir: __dirname, timeout: 30 * 1000 });
+
+const baseTest = folio.newTestType()
+
+export const test = baseTest.extend({
+  beforeAll: () => {
+    test.expect.extend({
```

Comment:
You can extend the `folio.expect` object - it is the same for all tests. This way it should be much cleaner.

Reply:
good catch!